package.json

```
@@ -6,9 +6,8 @@
   },
   "license": "Apache-2.0",
   "scripts": {
+    "install": "yarn check --integrity || yarn install --pure-lockfile --frozen-lockfile --ignore-scripts --skip-integrity-check --prefer-offline --link-duplicates",
```

`yarn check` [will be deprecated](https://classic.yarnpkg.com/en/docs/cli/check/#toc-yarn-check) in yarn 2.0 and can be replaced by `--check-files`.
I think overriding the default script is going to cause us some painful moments.

`frozen-lockfile` is redundant, it already includes `pure-lockfile`.
`ignore-scripts` is a potential constraint, it's better to just not to call it `install`.
`skip-integrity-check` doesn't make sense anymore (because of point made at line 1)
```suggestion
    "install:safe": "yarn install --frozen-lockfile --ignore-scripts --check-files --prefer-offline --link-duplicates",
```

---

What's actually the reasoning behind `ignore-scripts`?